Makefile

```diff
@@ -108,6 +108,10 @@ PL011_GENERIC_UART		:= 0
 ENABLE_PMF			:= 0
 # Flag to enable PSCI STATs functionality
 ENABLE_PSCI_STAT	:= 0
+# Whether code and read-only data should be put on separate memory pages.
+# The platform Makefile is free to override this value.
+SEPARATE_CODE_AND_RODATA	:= 0
+
```

Comment:
What's the argument against making this behavior as the default? (i.e. SEPARATE_CODE_AND_RODATA = 1 by default) I understand that this change fixes a security hole, so all platforms will benefit if we make this as the default option.